atd-vze/src/views/Crashes/Crash.js

```diff
@@ -279,6 +280,11 @@ function Crash(props) {
       ) : (
         <div></div>
       )}
+      <Row>
+        <Col>
+          <Notes data={data} {...props} />
```

Comment:
Hey Mike, thank you for these comments! I think you are absolutely correct that passing data as a prop is unnecessary there. And I also agree that it would make the code much more readable to only pass along the one crashID props. Haha you are correct in assuming I was basing mine on the existing code from a few lines down but I really appreciate you pointing these things out, I always want my code to be as readable/clear as possible and your comments have helped me to understand my own code even a little more! Gonna implement those changes :)